src/components/common/changeTransactionArray.js

```diff
@@ -0,0 +1,15 @@
+// import {
+//   getTransactionEntries,
+//   postNewTransactionEntry,
+// } from '../utils/services'
+// import React from 'react'
+
+export function monthlyBudget(transactions) {
```

Comment:
I find your naming a little confusing. changeTransactionArray suggest me you change an existing array yet your function is named monthlyBudget. If you change you array and your monthly budget here after a delete event you could name the function changeMonthlyBudget or updateMo(...). But I might misunderstand it here